components/analytics/overview/error-distribution-card.tsx

```
@@ -136,32 +308,144 @@ export function ErrorDistributionCard({
+      return `${baseColor}${Math.round(opacity * 255).toString(16).padStart(2, '0')}`
+    }
+    return scale(name) as string
   }, [errorDistributionData])
 
   // Chart configuration
   const chartConfig = useMemo(() => {
-    return errorDistributionData.reduce(
-      (acc, item) => {
-        acc[item.name] = {
+    return filteredData.reduce(
+      (acc, item: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)